client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dashboard-v2.tsx

```diff
@@ -247,11 +258,28 @@ export default function SitesDashboardV2() {
 					</div>
 				</div>
 				<div className="sites-overview__content">
-					<SitesDataViews
-						data={ data }
-						isLoading={ isLoading }
-						onViewChange={ onSitesViewChange }
-					/>
+					<DashboardDataContext.Provider
+						value={ {
+							verifiedContacts: {
+								emails: verifiedContacts?.emails ?? [],
+								phoneNumbers: verifiedContacts?.phoneNumbers ?? [],
+								refetchIfFailed: () => {
+									if ( fetchContactFailed ) {
+										refetchContacts();
+									}
+									return;
+								},
+							},
+							products: products ?? [],
+							isLargeScreen: isLargeScreen || false,
+						} }
+					>
```

Comment:
This is to fix the `Get Score` issue, right? Some generic questions:

- do we need all the fields? I'm not sure about `verifiedContacts`. I think `products` and probably `isLargeScreen` is okay. Do you who is using `verifiedContacts` in the old component?

Reply:
It seems that the `downtime-monitoring` components use it.

Reply:
Thanks, I agree.